src/vinylcontrol/vinylcontrol.cpp

```diff
@@ -21,8 +22,8 @@ VinylControl::VinylControl(ConfigObject<ConfigValue> * pConfig, QString group)
     trackSamples        = new ControlObjectThread(group, "track_samples");
     trackSampleRate     = new ControlObjectThread(group, "track_samplerate");
     vinylSeek           = new ControlObjectThread(group, "vinylcontrol_seek");
-    controlScratch      = new ControlObjectThread(group, "vinylcontrol_rate");
-    rateSlider          = new ControlObjectThread(group, "rate");    //Range -1.0 to 1.0
+    m_pVCRate = new ControlObjectSlave(group, "vinylcontrol_rate");
```

Comment:
I know, but you've only change two of the member variables, so now it's a confusing mix between old and new style.  Yes I plan to fix it, but you're combining cleanup work with bugfix work.  I really want to insist that this PR just fix the bug and not worry about cleaning up vinylcontrol -- that's something that can wait until after release.  Otherwise the commit history will be confusing later.

Of course it's sort of too late for that because it's already in the commit history -- this is one reason I really don't like our our unsquashed commit system, it makes things really hard to read :(